### ISSUE TITLE
[Data@TAMU] Example dataset page

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#13]
**URL:** https://data-pre.library.tamu.edu/entities/dataset/166dfef3-5439-4dad-b92f-45289a2ad3ba
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:** Broken ARIA menu
**Notes:** There is a menubar with no items within it: `<div _ngcontent-dspace-angular-c2948902503="" role="menubar" class="dso-edit-menu d-flex">`
